Jenkinsfile

```diff
@@ -0,0 +1,62 @@
+@Library('jenkins-libs') _
```

Comment:
Can we use [dockerPlainImagePipeline](https://github.com/CrowdFlower/jenkins-libs/blob/master/vars/dockerPlainImagePipeline.groovy)?


```
dockerPlainImagePipeline {
  image_name = 'odd-collector'
}